content/automate/ManageIQ/System/Event/EmsEvent/LenovoXclarity.class/__class__.yaml

```diff
@@ -0,0 +1,433 @@
+---
+object_type: class
+version: 1.0
+object:
+  attributes:
+    description: 
+    display_name: 
+    name: LenovoXclarity
```

Comment:
Event class name should match what is defined by ```ems_event.source``` in [event parser](https://github.com/ManageIQ/manageiq-providers-lenovo/blob/dad0108e6cf54d7a7f2e2d13784ab4c7c42cbafc/app/models/manageiq/providers/lenovo/physical_infra_manager/event_parser.rb#L5).